assignments/assignment01.rb

```diff
@@ -67,8 +103,33 @@ def my_reverse(s)
 
 # Write a method `palindrome?`
 # that determines whether a string is a palindrome
+  
 def palindrome?(s)
-  # your implementation here
+  
+  # => PSEUDO CODE
```

Comment:
You are approaching this too low level. You don't need to think it terms of looking at individual characters. You simply want to know if your string is the same as the reverse of the string.

One additional thing you need to do is strip out spaces and punctuation marks. You aren't doing that anywhere.